jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -36,6 +37,26 @@ def test_load_spectrum1d(self):
 
         assert isinstance(data, Spectrum1D)
 
+    def test_load_hdulist(self):
+        # Create a fake fits file with a 1D spectrum for testing.
+        primary_header = fits.Header({'TELESCOP': 'Fake Telescope'})
+        primary_hdu = fits.PrimaryHDU(header=primary_header)
+
+        # Image extension HDU with a 1D spectrum
+        wavelength = np.linspace(5000, 6000, 100)
+        flux = np.random.random(100)
```

Comment:
No need to waste random number computation for this test since the values don't matter.
```suggestion
        flux = np.ones(100)
```

Reply:
Just updated the Change log and the test in the most recent commit, thank you!